src/main/target/common_post.h

```diff
@@ -625,3 +625,15 @@ extern uint8_t __config_end;
 #ifdef USE_GPS_LAP_TIMER
 #define USE_CMS_GPS_LAP_TIMER_MENU
 #endif
+
+// Enable PINIO by default if any PIN is defined
+#if defined(PINIO1_BOX) || defined(PINIO2_BOX) || defined(PINIO3_BOX) || defined(PINIO4_BOX)
+#ifndef USE_PINIO
+#define USE_PINIO
+#endif
+#endif
+
+#if defined(USE_PINIO)
+#define USE_PINIOBOX
```

Comment:
Can you add a check for already defined for these?

Reply:
Done.